packages/account/src/BaseAccount.ts

```diff
@@ -0,0 +1,171 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { ethers, BigNumber, BigNumberish, Signer, BytesLike } from 'ethers'
+import { ISmartAccount } from './interfaces/IBaseAccount'
+import { defaultAbiCoder, keccak256, arrayify } from 'ethers/lib/utils'
+import { UserOperation, ChainId } from '@biconomy/core-types'
+import { calcPreVerificationGas, DefaultGasLimits } from './utils/Preverificaiton'
+import { packUserOp } from '@biconomy/common'
+import { DEFAULT_CALL_GAS_LIMIT, DEFAULT_VERIFICATION_GAS_LIMIT, DEFAULT_PRE_VERIFICATION_GAS, EIP1559_UNSUPPORTED_NETWORKS } from './utils/Constants'
+import { IBundler, Bundler } from '@biconomy/bundler'
+import { IPaymasterAPI } from '@biconomy/paymaster'
+import {
+    EntryPoint_v100,
+    SmartAccount_v100,
+} from '@biconomy/common'
+import { SmartAccountConfig } from './utils/Types'
+export const DEFAULT_USER_OP: UserOperation = {
+    sender: ethers.constants.AddressZero,
+    nonce: ethers.constants.Zero,
+    initCode: ethers.utils.hexlify("0x"),
+    callData: ethers.utils.hexlify("0x"),
+    callGasLimit: DEFAULT_CALL_GAS_LIMIT,
+    verificationGasLimit: DEFAULT_VERIFICATION_GAS_LIMIT,
+    preVerificationGas: DEFAULT_PRE_VERIFICATION_GAS,
+    maxFeePerGas: ethers.constants.Zero,
+    maxPriorityFeePerGas: ethers.constants.Zero,
+    paymasterAndData: ethers.utils.hexlify("0x"),
+    signature: ethers.utils.hexlify("0x"),
```

Comment:
this is wrong signature length should be 65. or use constant valid signature as mock signature?

Reply:
just assigning mock value

Reply:
at least repeat 0 65 times...
or some valid ecdsa signature (any signer)